test/parallel/test-stdin-script-child.js

```diff
@@ -22,11 +22,11 @@ child.stderr.on('data', function(c) {
 });
 
 child.on('close', function(c) {
```

Comment:
We need butter documentation for `common.mustCall()` for new contributors. I've just opened https://github.com/nodejs/node/pull/10390 to slightly expand what we have.

@emanuelbuholzer If you need a little bit more information about `common.mustCall()`, it's at https://github.com/Trott/io.js/blob/bd2ae52fc3fbb9ce6c0e8c79ff5e67f8a742847b/test/README.md#mustcallfn-expected for now. You can also look at how it is used in other tests. And you can also ask questions here or on the #node-dev channel on Freenode IRC.